ci/Dockerfile

```diff
@@ -5,6 +5,9 @@ ARG GINKGO_VERSION=latest
 ARG GOLANGCILINT_VERSION=latest
 RUN apt-get update && apt-get install -y libpcap-dev && rm -rf /var/lib/apt/lists/*
 
+RUN curl -fsSL https://deb.nodesource.com/setup_current.x | sudo -E bash - && \
```

Comment:
Should we consider pulling in the data from `node:20.x`/ `node:current` or similar?

```Dockerfile
ARG NODE_VERSION=current
FROM node:${NODE_VERSION} as nodejs
COPY --from=nodejs /usr/bin/node
```
(along the lines of this, not tested, just as idea)

Reply:
See also https://github.com/moby/moby/issues/45419#issuecomment-1525256205

Reply:
we don't just need node but npx to run semantic-release. So I'm not sure this way would work or what else would be needed to be copied. For go it's easy because you only have one binary.
So unless I'd have to go a different way, I'd say we keep it as it is now.

This is also the official way to install nodejs on ubuntu, see https://nodejs.org/en/download/package-manager#debian-and-ubuntu-based-linux-distributions